src/const_game.h

```diff
@@ -0,0 +1,45 @@
+// hunger thresholds and messages
```

Comment:
In theory so that any constexpr expressions they're fed to can be evaluated at runtime.  In practice, I'm skeptical of that happening to any significant degree.  OTOH, it doesn't hurt, it's "const with potentially beneficial side effects".